src/main/java/org/spongepowered/api/data/DataHolder.java

```diff
@@ -24,46 +24,310 @@
  */
 package org.spongepowered.api.data;
 
-import org.spongepowered.api.data.manipulator.DataManipulator;
+import org.checkerframework.checker.nullness.qual.NonNull;
```

Comment:
Just curious, where did you get this checker framework annotation from? I fear my change in #2001 might clash with yours.

Reply:
Transient dependency from configurate, most likely.